docs/source/getting-started/assessing-utility/accuracy.ipynb

```diff
@@ -0,0 +1,374 @@
+    "\n",
+    "However, this approach isn't concrete since interpretations can vary widely on scale, type of visualization, and other factors. A more objective way to evaluate differentially private outputs is by using the `accuracy` method. \n",
+    "\n",
+    "With synthetic data a comparison with the \"true\" value is possible. However, this is not possible with real data. If the analysis plan is influenced by the actual data, it's a privacy leak. \n",
+    "\n",
+    "\n",
+    "In this notebook, we will: \n",
+    "- Explain the purpose and the application of the `accuracy` method. \n",
+    "- Clarify how to interpret the results of the `accuracy` method. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import opendp.prelude as dp\n",
+    "import polars as pl \n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np \n",
+    "import pandas as pd \n",
+    "\n",
+    "dp.enable_features(\"contrib\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+      "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
+      "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
+      "100 5933k  100 5933k    0     0  1035k      0  0:00:05  0:00:05 --:--:-- 1099k:06  0:00:02  0:00:04 1047k\n",
+      "Archive:  data.zip\n",
+      "  inflating: sample_FR_LFS.csv       \n",
+      "  inflating: __MACOSX/._sample_FR_LFS.csv  \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Fetch and download data. \n",
+    "!curl \"https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip?raw=true\" -L -o data.zip\n",
+    "!unzip data.zip\n",
+    "df = pl.scan_csv(\"sample_FR_LFS.csv\", ignore_errors=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "context = dp.Context.compositor(\n",
+    "    data=df,\n",
+    "    privacy_unit=dp.unit_of(contributions=36),\n",
+    "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+    "    split_evenly_over=3,\n",
+    "    margins={\n",
+    "        (\"YEAR\", \"QUARTER\",): dp.Margin(public_info=\"keys\", max_partition_length=60_000_000, max_partition_contributions=1),\n",
+    "        (): dp.Margin(max_partition_length=60_000_000),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use the same query we began with in [Part 1](http://example.com/TODO): a count query for each year and quarter."
```

Comment:
- "with in" to "within"
- Fix the URL

Reply:
Since all the notebooks are in separate PRs, the thing this would point to isn't in this branch. I've marked this notebook as orphan in the metadata, and there will be a follow-up PR to make these all public and link into the table-of-contents.